install_files/ansible-base/roles/tails-config/templates/extension.js.in

```diff
@@ -91,7 +91,7 @@ class Indicator extends PanelMenu.Button {
             Util.trySpawnCommandLine(`keepassxc`);
         });
 
-        let filebrowser = new PopupMenu.PopupMenuItem(_('Open File Browser'));
+        let filebrowser = new PopupMenu.PopupMenuItem('Open File Browser');
```

Comment:
Why wouldn't we mark this string for translation?

Reply:
We definitely would, although for some odd reason it was originally the _only_ string marked for translations, so I made that tweak to get everything to a consistent baseline.

I figured out why it was breaking things, and pushed a new commit this morning which should fix the translations and make all the strings properly translatable.